src/OpenTelemetry.Collector.Dependencies/OpenTelemetry.Collector.Dependencies.csproj

```diff
@@ -1,12 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <Description>OpenTelemetry collector for calls to dependencies</Description>
     <PackageTags>$(PackageTags);distributed-tracing;HttpClient</PackageTags>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\OpenTelemetry\OpenTelemetry.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net461'" />
```

Comment:
why do we need `System.Web` reference here? `System.Web` may look like a surprise for worker roles  using HttpWebRequest

Reply:
Good call, I will fix.